models/model.js

```diff
@@ -15,3 +15,21 @@ exports.selectArticles = () => {
       return articles;
     });
 };
+
+exports.selectArticleById = (id) => {
+  return db
+    .query("SELECT * FROM articles WHERE article_id = $1;", [id])
+    .then(({ rows }) => {
```

Comment:
This and the next ".then" block can be done in a single .then, but it is not a requirement, no need to refactor if you do not want to 🙂